### ISSUE TITLE
hyperparameters and model configs for computing the metrics in the OpenP5 paper

### ISSUE THREAD

**Issue by liyang2019** [agiresearch/OpenP5#34]
Hello,

Wondering if there is a place that we can find the exact hyperparameters (id vocab size, num clusters, learning rate, etc.) and model config (num_layers, hidden size, etc.) to reproduce the metrics in the OpenP5 paper (https://arxiv.org/pdf/2203.13366.pdf)?

Thanks!
